packages/demo-app/vite.config.ts

```diff
@@ -1,15 +1,22 @@
-import { mergeConfig, type UserConfig } from 'vite';
+import react from '@vitejs/plugin-react';
```

Comment:
Just curous, as it seems it works even without this React plugin. Interesting..

Reply:
react plugin is for fast refresh. nothing needed to make it "work"